src/docs/content/observability/observability-example/_index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Observability Example"
+date: 2020-10-26 T16:04:15-06:00
+draft: false
+weight: 110
+showChildren: true
+---
+
+## Introduction
+
+In this section we will try to understand Observability on Oracle Cloud Infrastructure (OCI) by simulating a failure scenario on MuShop application.
+
+![Failure-scenario](../images/observability-scenario.png)
```

Comment:
If is observability scenario, image name should follow same naming.

Reply:
changed all the image names.